interceptor/main.go

```diff
@@ -101,6 +110,13 @@ func main() {
 		return err
 	})
 
+	errGrp.Go(func() error {
+		defer ctxDone()
+		err := configMapInformer.Start(ctx)
+		lggr.Error(err, "config map watcher failed")
+		return err
+	})
+
```

Comment:
I think it would make sense to start the `configMapInformer` from inside of the `StartConfigMapRoutingTableUpdater` so that all of the routing table-related functionality is in one place. What do you think?

Reply:
You're right. I will move the `configMapInformer.Start()` into `StartConfigMapRoutingTableUpdater`.